derive/src/utils.rs

```diff
@@ -42,8 +43,11 @@ fn find_meta_item<'a, F, R, I>(itr: I, pred: F) -> Option<R> where
 	}).next()
 }
 
-pub fn index(v: &Variant, i: usize) -> TokenStream {
-	// look for an index in attributes
+/// Look for a `#[scale(index = $int)]` attribute on a variant. If no attribute
+/// is found, fall back to the discriminant or just the variant index.
+/// Returns a `TokenStream`.
```

Comment:
I usually prefer not to write anything, if we don't add any information on the type itself
```suggestion
```